geotrek/common/templatetags/geotrek_tags.py

```diff
@@ -61,3 +63,26 @@ def duration(value):
         final_duration = _("%s min") % duration.minute
 
     return final_duration
+
+
+@register.simple_tag
+def restricted_area_types():
```

Comment:
You should place these tags in geotrek.zoning.template_tags. This will be used in georiviere, which use zoning but not common. Name file zoning_tags.py